pkg/ddc/alluxio/transform.go

```diff
@@ -202,9 +202,60 @@ func (e *AlluxioEngine) transformCommonPart(runtime *datav1alpha1.AlluxioRuntime
 
 	value.Tieredstore.Levels = levels
 
-	value.Journal = Journal{
-		VolumeType: "emptyDir",
-		Size:       "30Gi",
+	// value.Metastore = Metastore{
```

Comment:
How about moving this logic into a sperate method.  And add testing for it. Thanks.

Reply:
SGTM